lib/srv/db/server.go

```diff
@@ -465,13 +482,26 @@ func (s *Server) startDatabase(ctx context.Context, database types.Database) err
 	if err := s.cfg.CloudUsers.Setup(ctx, database); err != nil {
 		s.log.WarnContext(ctx, "Failed to setup users.", "database", database.GetName(), "error", err)
 	}
+	// Start database object importer.
+	if err := s.cfg.DatabaseObjects.StartImporter(ctx, database); err != nil {
+		// special handling for "not implemented" errors; these are very likely to occur and aren't as interesting.
+		if trace.IsNotImplemented(err) {
+			s.log.DebugContext(ctx, "Database object importer not implemented.", "database", database.GetName())
+		} else {
+			s.log.WarnContext(ctx, "Failed to start database object importer.", "database", database.GetName(), "error", err)
```

Comment:
Are we still getting this when postgres databases are not configured with auto-user provisioning?

Reply:
No, implementation-wise `NotImplemented` is only returned for unsupported protocols.

Reply:
> No, implementation-wise `NotImplemented` is only returned for unsupported protocols.

a postgres database without admin user will return BadParameter and come here?

Reply:
Yes, this will give a warning. Do you think we should handle that case separately?

Reply:
i mean not setting admin user is a common case, not an error case. we shouldn't error/warn in that case, or maybe we shouldn't even start importer when admin user is not set.

Reply:
I took it from the perspective of answering the question of "why does the importer doesn't work for my database", but you are right, this doesn't deserve a treatment of an error. I'll add handling for this.

Reply:
We will now get this sort of message (debug verbosity):
```
2024-07-12T15:20:30+02:00 DEBU [DB:SERVIC] Database object importer cannot be started due to disabled fetcher reason:no admin user configured database:mypg db/server.go:491
```